cdi/embedded/src/test/resources/infinispan.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <infinispan
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-      xsi:schemaLocation="urn:infinispan:config:9.1 http://www.infinispan.org/schemas/infinispan-config-9.1.xsd"
```

Comment:
It crossed my mind, the downside is that IDE wwill not be able to validate the XML properly and potentially the XML will be invalid, WDYT?